src/rpm/headers/header.rs

```diff
@@ -575,7 +575,6 @@ pub struct FileOwnership {
 }
 
 /// Declaration what category this file belongs to
-/// @todo must be bitflags
```

Comment:
I'm not sure that using bitflags in this particular spot makes sense, since AFAIK the file can only belong to one category.